src/jit/rationalize.cpp

```diff
@@ -526,70 +503,54 @@ void copyFlags(GenTree *dst, GenTree *src, unsigned mask)
 
 //--------------------------------------------------------------------------------------
 // RewriteTopLevelComma - split a top-level comma into two top level statements.
-//                                returns (as out params) the two new locations
+//                                returns the location of the new statement.
 //--------------------------------------------------------------------------------------
```

Comment:
The comment's a little confusing, since it says it splits into *two* statements, but returns the loc of "the" new statement. But if there are 2, which is "the" statement? Presumably, it means "the next statement to process"